src/offline/syncUpdatesWithBackend.ts

```diff
@@ -419,6 +419,16 @@ const updateIdOnEntityRequest = (
                 );
                 offlineUpdate.entityId = newId;
             }
+        } else if (offlineUpdate.url.startsWith('PunchListItem/AddComment?')) {
+            console.log(offlineUpdate.entityId);
+            const oldId = offlineUpdate.entityId;
+            if (oldId) {
+                offlineUpdate.url = offlineUpdate.url.replace(
```

Comment:
The get comments url includes punch item id, and if the post url did the same then you could hande this in this way. However, the opst comment stores the entity id (punch id) in the bodydata instead, so here you should handle the update of the entity id the same way as it is done for punchlistitem/clear and the others.